charts/agent/templates/daemonset.yaml

```diff
@@ -249,6 +249,10 @@ spec:
               name: sysdig-agent-config
             - mountPath: /opt/draios/etc/kubernetes/secrets
               name: sysdig-agent-secrets
+            {{- if (include "agent.httpProxyCredentials" .) }}
```

Comment:
We should probably perform the same changes in `charts/agent/templates/deployment.yaml`

Reply:
uh thanks, I was missing that! 🙏